packages/server/lib/browsers/chrome.ts

```diff
@@ -119,6 +120,9 @@ const DEFAULT_ARGS = [
   // write shared memory files into '/tmp' instead of '/dev/shm'
   // https://github.com/cypress-io/cypress/issues/5336
   '--disable-dev-shm-usage',
+
+  // enable precise memory info so performance.memory returns more accurate values
+  '--enable-precise-memory-info',
```

Comment:
should/can we enable this based on the experimental flag?

Reply:
I believe we could, I don't think it's really worth it though.

Reply:
Theres no cost to enabling the flag?

Reply:
The memory information is updated more often but I haven't noticed any impact on performance.